ui/app/components/pages/unlock-page/unlock-page.container.js

```diff
@@ -1,23 +1,22 @@
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { compose } from 'recompose'
-
-const {
+import {
   tryUnlockMetamask,
   forgotPassword,
   markPasswordForgotten,
-} = require('../../../actions')
-
+} from '../../../actions'
 import UnlockPage from './unlock-page.component'
 
-const mapStateToProps = state => {
```

Comment:
Why do you prefer `function mapStateToProps (state) {` to `const mapStateToProps = state => {`?